.github/workflows/onPushToMain.yml

```diff
@@ -0,0 +1,48 @@
+name: Push to main
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  tests:
+      uses: salesforcecli/github-workflows/.github/workflows/unitTest.yml@main
+  release:
+    runs-on: ubuntu-latest
+    container:
+      image: node:lts
+    needs:
+    - tests
+    steps:
+    - uses: actions/download-artifact@v2
+    - uses: actions/checkout@v3
+    - name: Configuring GitHub
+      run: |-
+        git config credential.helper 'cache --timeout=120'
+        git config user.email "$GH_EMAIL"
+        git config user.name "Release Bot"
+    - name: Bump package version
```

Comment:
Instead of this step, can you add in the conventional changelog action?

https://github.com/forcedotcom/salesforcedx-templates/blob/777e382e78f457d631c3a55fe3956ee2763aac90/.github/workflows/onPushToMain.yml#L20

You'll want to use the whole step from templates, except for the fallback-version set that to the current version of the LSP (./lerna.json version). 

Then in your bump-lerna-version script, insert lines 26, 27, 28 (yarn, yarn bump-versions with the version from the changelog action (see templates bump-version script), and git add . )  

And remove the "Tag the release and push to repo step", since that will be covered with the changelog action.